modules/installation/pages/uyuni-proxy-setup.adoc

```diff
@@ -0,0 +1,249 @@
+
+Proxy Version to Activate::
+Normally, the correct value should be offered as a default.
+
+Traceback Email::
+An email address where to report problems.
+
+Use SSL::
+For safety reasons, press ``Y``.
+
+Do You Want to Import Existing Certificates?::
+Answer ``N``.
+This ensures using the new certificates that were copied previously from the {productname} server.
+
+Organization::
+The next questions are about the characteristics to use for the SSL certificate of the proxy.
+The organization might be the same organization that was used on the server, unless of course your proxy is not in the same organization as your main server.
+
+Organization Unit::
+The default value here is the proxy's hostname.
+
+City::
+Further information attached to the proxy's certificate.
+
+State::
+Further information attached to the proxy's certificate.
+
+Country Code::
+In the [guimenu]``country code`` field, enter the country code set during the {productname} installation.
+For example, if your proxy is in the US and your {productname} is in DE, enter `DE` for the proxy.
++
+
+[NOTE]
+====
+The country code must be two upper case letters.
+For a complete list of country codes, see https://www.iso.org/obp/ui/#search.
+====
+
+Cname Aliases (Separated by Space)::
+Use this if your proxy can be accessed through various DNS CNAME aliases.
+Otherwise it can be left empty.
+
+CA Password::
+Enter the password that was used for the certificate of your {productname} Server.
+
+Do You Want to Use an Existing SSH Key for Proxying SSH-Push Salt Minion?::
+Use this option if you want to reuse a SSH key that was used for SSH-Push Salt clients on the server.
+
+Create and Populate Configuration Channel rhn_proxy_config_1000010001?::
+Accept default ``Y``.
+
+SUSE Manager Username::
+Use same user name and password as on the {productname} server.
+
+If parts are missing, such as CA key and public certificate, the script prints commands that you must execute to integrate the needed files.
+When the mandatory files are copied, run [command]``configure-proxy.sh`` again.
+If you receive an HTTP error during script execution, run the script again.
+
+[command]``configure-proxy.sh`` activates services required by {productname} Proxy, such as [systemitem]``squid``, [systemitem]``apache2``, [systemitem]``salt-broker``, and [systemitem]``jabberd``.
+
+To check the status of the proxy system and its clients, click the proxy system's details page on the {webui} (menu:Systems[Proxy], then the system name).
+[guimenu]``Connection`` and [guimenu]``Proxy`` subtabs display various status information.
+
+
+
+[[uyuni-proxy-setup-pxe]]
+== Enable PXE Boot
+
+
+
+[[proxy.pxe.sync]]
+=== Synchronize Profiles and System Information
+
+To enable PXE boot through a proxy, additional software must be installed and configured on both the {productname} Proxy and the  {productname} Server.
+
+. On the {productname} Proxy install [package]#susemanager-tftpsync-recv#:
```

Comment:
```suggestion
. On the {productname} Proxy, install [package]#susemanager-tftpsync-recv#:
```

Reply:
Not sure about the pound/hash signs?

Reply:
> Not sure about the pound/hash signs?

We have those at several places...  It's adoc ;)